docs/tutorials.rst

```diff
@@ -57,13 +57,3 @@ Timeseries Explainers
    :glob:
 
    tutorials/timeseries/*
-
-
-Learning-To-Ranking Explainers
```

Comment:
@yangwenzhuo08 
Would the documentation display the tutorial if this is removed?